src/listconf.c

```diff
@@ -323,13 +319,14 @@ static void listconf_list_build_info(void)
 	printf("times(2) CLK_TCK is %ld\n", clk_tck);
 #endif
 #if defined (__MINGW32__) || defined (_MSC_VER)
-	printf("Using clock(3) for timers, claimed resolution %ss, observed %ss\n",
-	       human_prefix_small(1.0 / CLOCKS_PER_SEC),
-	       human_prefix_small(1.0 / sm_cPrecision));
+	printf("Using clock(3) for timers, resolution %ss\n", human_prefix_small(1.0 / CLOCKS_PER_SEC));
 #else
 	printf("Using times(2) for timers, resolution %ss\n", human_prefix_small(1.0 / clk_tck));
 #endif
-	printf("HR timer claimed resolution %ss, observed %ss\n", human_prefix_small(1.0 / sm_HRTicksPerSec), human_prefix_small(1.0 / sm_hrPrecision));
+	int latency;
+	uint64_t precision = timer_stats(&latency);
+
+	printf("HR timer: " HR_CLOCK ", resolution %ss, %s %ss\n", human_prefix_small(1.0 / timer_ticks_per_sec), latency ? "latency" : "observed", human_prefix_small(precision / 1000000000.0));
```

Comment:
> It might bring num to just above 1.0, then round it to integer 1 - that's very little precision left. We need to fix it separately.

I see what you mean. I could add it as a separate commit of this PR.

Reply:
Moved to #4533